### PR TITLE
Allow EnergySystems Hizard to modify PV parameters and fix highlighting issues 

### LINKED ISSUE #663: EnergySystems Hizard cannot modify PV parameters
**Describe the bug**
In the EnergySystems hizard, currently, we cannot modify technology parameters of PV and SoalrThermal (cost, emissions, efficiency, ...). This should be made possible. Changed numbers should become highlighted green, as with the other technologies.

**Software**
 - Rhino Version 6
 - Hive Version 1.13.1

### PR DESCRIPTION
**Pull request by philipschulz**
## Issues

Closes #663

### CODE REVIEW

**Comment by christophwaibel** [comment]
it works! But there is one cosmetic bug. When clicking first into either one of the textboxes for `Module Efficiency`, `Specific Capital Cost`, or `Specific Embodied Emissions` and then clicking again into one of those textboxes, the value in the textbox for `Area` disappears. It re-appears upon re-opening the hizard:

image

image

File: src/Hive.IO/Hive.IO/Forms/Controls/SurfaceTechnologyProperties.cs [hunk 1]
```diff
@@ -29,6 +29,17 @@ public override ConversionTechPropertiesViewModel Conversion
...
+        ///     See [here](https://stackoverflow.com/a/3165330/2260) for why I'm not
+        ///     hooking the event up directly with the handler defined in the base class.
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        private new void Validating(object sender, CancelEventArgs e)
+        {
+            TextBox_Validating(sender, e);
+        }
+
```
Comment (christophwaibel) [inline src/Hive.IO/Hive.IO/Forms/Controls/SurfaceTechnologyProperties.cs:42]: not sure if the problem comes from here, but see [this comment](https://github.com/architecture-building-systems/hive/pull/666#issuecomment-963412851)

**Review by christophwaibel — CHANGES_REQUESTED** [review]
one cosmetic bug related to value for `Area` disappearing

**Comment by christophwaibel** [comment]
on my desktop, the hizard is narrower now. It won't properly display the "Module Type" drop down list. I actually like it smaller like that, but can you fix the displaying issue of the Module Type list?

image

It seems to auto-size-down to the width of the Source-Conversion-EndUse grid:

image

**Comment by christophwaibel** [comment]
Also, now, none of the parameters are showing up anymore. Hence, we can't change efficiency, etc., which was the original purpose of this PR ;-D  
See same screenshot above.

**Comment by philipschulz** [comment]
image
Uh, I'm not sure whats going on here, it look like this for me...

**Comment by christophwaibel** [comment]
hmm rebuilt on my home desktop and now also looks ok. Will merge for now and try on different computers

**Pull request merged**